src/database/models/Challenge/ExperiencedChallenge.ts

```diff
@@ -0,0 +1,58 @@
+import { DataTypes, Sequelize, Model } from 'sequelize';
+import { ChallengeCardBase } from './ChallengeCard';
+
+class ExperiencedChallengeCard
+  extends Model<ChallengeCardBase>
+  implements ChallengeCardBase
+{
+    user_id: string;
+    challengeOne: string;
+    challengeTwo: string;
+    challengeThree: string;
+    challengeFour: string;
```

Comment:
I think once challenges exist in the database under a "challenge bank", we can most likely combine all the different challenge tables to one single `UserChallenge` and some additional changes. 

Here's an idea of the structure:

`DiscordUser`:
```
...
user_challenge: number (non-completed UserChallenge id)
```

`Challenge`:
```
id: number (PK)
description: string
difficulty: number
region_one: number
region_two: number
```

`UserChallenge`:
```
id: number (PK)
user_id: string
difficulty: number
challenge_one: id
challenge_two: id
challenge_three: id
challenge_four: id | null
challenge_five: id | null
reroll_count: number
completed: boolean
```

`UserChallengeSubmission`:
```
id: number (PK)
user_challenge: number
challenge: number
proof: string
```

This would also allow us to let a user view their historically completed challenges difficulties. There might be some further improvements that could be made with restraints and associations.